python/nx-cugraph/nx_cugraph/interface.py

```diff
@@ -62,19 +71,32 @@ def key(testpath):
                 return (testname, frozenset({classname, filename}))
             return (testname, frozenset({filename}))
 
+        use_compat_graph = (
+            _nxver < (3, 3) or nx.config.backends.cugraph.use_compat_graphs
+        )
+        fallback = use_compat_graph or nx.utils.backends._dispatchable._fallback_to_nx
+
         # Reasons for xfailing
+        # For nx version <= 3.1
         no_weights = "weighted implementation not currently supported"
         no_multigraph = "multigraphs not currently supported"
+        # For nx version <= 3.2
+        nx_cugraph_in_test_setup = (
+            "nx-cugraph Graph is incompatible in test setup in nx versions < 3.3"
+        )
```

Comment:
This is surprising since I figured nxcg.Graph is now a superset of the old nxcg.Graph. Why isn't it compatible?

Reply:
I don't recall; something changed in the dispatch machinery. This is a breadcrumb that can be investigated further if desired; all other tests pass (or behave as expected) when I ran with earlier versions of NetworkX.